integration/fuse/bin/alluxio-fuse

```diff
@@ -47,6 +47,7 @@ set_java_opt() {
     -server
     -Xms1G
     -Xmx1G
+    -XX:MaxDirectMemorySize=4g
```

Comment:
Can users easily modify this (ie. by exporting `ALLUXIO_FUSE_JAVA_OPTS` in alluxio-env.sh)?

Reply:
Looks like it works!